projects/kserve/testing/config.yaml

```
@@ -192,6 +192,12 @@ ci_presets:
     matbench.lts.horreum.test_name: null
     matbench.lts.opensearch.index: rhoai-kserve-multi-light
 
+  multi_model_longevity:
+    extends: [multi_model_gating]
+    tests.e2e.longevity.enabled: true
+    tests.e2e.longevity.repeat: 3
+    tests.e2e.longevity.delay: 300
```

can you sleep that in two:
```suggestion
  multi_model_longevity:
    extends: [multi_model_gating]
    tests.e2e.longevity.enabled: true

  multi_day_longevity:
    tests.e2e.longevity.repeat: 3
    tests.e2e.longevity.delay: ...
```
we'll adjust this one when we know exactly what we want test, maybe once every 12h for 36h